docs/python/environments.md

```diff
@@ -4,39 +4,42 @@ Area: python
+Working in the global environment is an easy way to get started. In time, that environment will become crowded with the different packages, making it difficult to thoroughly test an application.
 
-Although working in the global environment is an easy way to get started, that environment will, over time, become cluttered with many different packages that you've installed for different projects. Such clutter makes it difficult to thoroughly test an application against a specific set of packages with known versions, which is exactly the kind of environment you'd set up on a build server or web server.
+### Virtual environments
 
-For this reason, developers often create a **virtual environment** for a project. A virtual environment is a subfolder in a project that contains a copy of a specific interpreter. When you activate the virtual environment, any packages you install are installed only in that environment's subfolder. When you then run a Python program within that environment, you know that it's running against only those specific packages. Be aware that if you're not using a **virtual environment**, and you have multiple versions of Python installed and set in the `path` environment variable, you might need to specify the Python interpreter to use in the terminal for installing packages to the global environment.
+To prevent such clutter, developers often create a **virtual environment** for a project. A virtual environment is a subfolder in a project that contains a copy of a specific interpreter. When you activate the virtual environment, any packages you install are installed only in that environment's subfolder. When you then run a Python program within that environment, you know that it's running against only those specific packages. Be aware that if you're not using a **virtual environment**, and you have multiple versions of Python installed and set in the `path` environment variable, you might need to specify the Python interpreter to use in the terminal for installing packages to the global environment.
 
 > **Note**: While it's possible to open a virtual environment folder as a workspace, doing so is not recommended and might cause issues with using the Python extension.
 
+### Python environment tools
+Once you activate your virtual environment, you’ll need to identify how to manage it and its accompanying packages. The following table explains how to use these Python environments:
+
+|   Tool         | Definition and Purpose    |
+|   ---          | --- |
+|   pip          | Manages packages and is installed with Python by default.      |
+|   virtualenv   | Manages virtual environments and is installed with Python by default.   |
```

Comment:
I don't think virtualenv is installed with Python by default. venv is installed in a lot of them but also not all, AFAIK

Reply:
Updated this as well. Let me know if you think this table has the potential to provide users some clarity. I'm also happy to remove if not!